e2e/src/api/specs/activity.e2e-spec.ts

```diff
@@ -245,11 +271,11 @@ describe(`${ActivityController.name} (e2e)`, () => {
     });
 
     it('should return a 200 for a duplicate like on the album', async () => {
-      const reaction = await api.activityApi.create(server, admin.accessToken, {
-        albumId: album.id,
-        type: ReactionType.LIKE,
-      });
-      const { status, body } = await request(server)
+      const [reaction] = await Promise.all([
```

Comment:
What's the reason for the promise.all here and a few lines below? Future-proofing?